docs/docs/how-to-deploy.mdx

```diff
@@ -124,7 +124,7 @@ jobs:
 
     - id: action_server
       name: Build an action server with a custom actions
-      uses: RasaHQ/action-server-gha@master
+      uses: RasaHQ/action-server-gha@main
       # Full list of parameters: https://github.com/RasaHQ/action-server-gha/tree/master#input-arguments
```

Comment:
Looks like there's a couple other uses of `master` in the file, could you change them too?